src/components/SelectItemModal/SelectItemModal.js

```diff
@@ -0,0 +1,101 @@
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Modal,
+  Paneset,
+  Pane,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import css from './SelectItemModal.css';
+
+const COLUMN_NAMES = [
+  'barcode',
+  'itemStatus',
+  'requestQueue',
+  'location',
+  'materialType',
+  'loanType',
+];
+
+const COLUMN_WIDTHS = {
+  barcode: '16%',
+  itemStatus: '16%',
+  requestQueue: '16%',
+  location: '16%',
+  materialType: '16%',
+  loanType: '16%',
+};
+
+const COLUMN_MAP = {
+  barcode: <FormattedMessage id="ui-checkin.selectItemModal.barcode" />,
+  itemStatus: <FormattedMessage id="ui-checkin.selectItemModal.itemStatus" />,
+  requestQueue: <FormattedMessage id="ui-checkin.selectItemModal.requestQueue" />,
+  location: <FormattedMessage id="ui-checkin.selectItemModal.location" />,
+  materialType: <FormattedMessage id="ui-checkin.selectItemModal.materialType" />,
+  loanType: <FormattedMessage id="ui-checkin.selectItemModal.loanType" />,
+};
+
+const formatter = {
+  itemStatus: item => item.status.name,
+  location: item => get(item, 'effectiveLocation.name', ''),
+  materialType: item => item.materialType.name,
+  loanType: item => (item.temporaryLoanType
+    ? get(item, 'temporaryLoanType.name', '')
```

Comment:
for new code I would suggest to use `?` operator instead of `get`, something like this method => `item.temporaryLoanType?.name ?? ''`

Reply:
Thanks, fixed